server/test/services/applicant/ApplicantDataTest.java

```diff
@@ -63,4 +63,14 @@ public void withFailedUpdates() {
     assertThat(data.getFailedUpdates()).isEqualTo(ImmutableMap.of(samplePath, "invalid_value"));
     assertThrows(IllegalStateException.class, () -> data.asJsonString());
   }
+
+  @Test
+  public void checkDateOfBirth() {
+    ApplicantData data = new ApplicantData();
+    String sampleDOB = "2022-01-05";
```

Comment:
suggestion (blocking): please see the style guide on case of abbreviations: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
I will update.